### ISSUE TITLE
Blop bop: focus on keys and play buttons

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#201]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/microphone/

Once focus reaches the keys on the piano  the focus is far too subtle. Should probably change the key colour. 

Also the play button doesn't have any focus styles.
